src/app/clusters/electrical-energy-measurement-server/tests/ClusterActions.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/testing/AttributeTesting.h>
+#include <app/data-model-provider/tests/ReadTesting.h>
+#include <app/data-model-provider/tests/WriteTesting.h>
+#include <app/reporting/ReportScheduler.h>
+#include <app/server-cluster/DefaultServerCluster.h>
+#include <app/server-cluster/testing/TestServerClusterContext.h>
+#include <cassert>
+#include <type_traits>
+
+// Helper function to write a value to an attribute.
+template <typename T>
+CHIP_ERROR WriteAttribute(chip::app::DefaultServerCluster & cluster, chip::AttributeId attributeId, const T & value)
+{
+    chip::Span<const chip::app::ConcreteClusterPath> paths = cluster.GetPaths();
+    assert(paths.size() == 1); // DefaultServerCluster guarantees a single path
+    const chip::app::ConcreteDataAttributePath path = { paths[0].mEndpointId, paths[0].mClusterId, attributeId };
+
+    chip::app::Testing::WriteOperation writeOperation(path);
+    chip::app::AttributeValueDecoder decoder = writeOperation.DecoderFor(value);
+    return cluster.WriteAttribute(writeOperation.GetRequest(), decoder).GetUnderlyingError();
+}
+
+// Helper function to invoke a command and return the result.
+template <typename T>
+std::optional<chip::app::DataModel::ActionReturnStatus> InvokeCommand(chip::app::DefaultServerCluster & cluster,
```

Comment:
use https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/testing/ClusterTester.h#L130 

Or generally enhance cluster testing infra instead of creating special-purpuse helpers for the cluster only.